lua/tokyonight/theme.lua

```diff
@@ -328,7 +340,24 @@ function M.setup(config)
     DiagnosticWarning = { fg = c.warning },
     DiagnosticInformation = { fg = c.info },
     DiagnosticHint = { fg = c.hint },
-
+	
+		LspFloatWinNormal =                     { bg = c.dark },
+		LspFloatWinBorder =                     { fg = c.border_highlight },
+		LspSagaBorderTitle =                    { fg = c.cyan },
+		LspSagaHoverBorder =                    { fg = c.blue },
+		LspSagaRenameBorder =                   { fg = c.green },
+		LspSagaDefPreviewBorder =               { fg = c.green },
+		LspSagaCodeActionBorder =               { fg = c.blue },
+		LspSagaFinderSelection =                { fg = c.bg_visual },
+		LspSagaCodeActionTitle =                { fg = c.blue1 },
+		LspSagaCodeActionContent =              { fg = c.purple },
+		LspSagaSignatureHelpBorder =            { fg = c.red },
+		ReferencesCount =                       { fg = c.purple },
```

Comment:
I'm a bit worried about adding these groups. They seem pretty broad. Are you sure there's no other plugins using these groups?

Reply:
I didn't find any plugins with these groups yet. (after a quick google search)